var/spack/repos/builtin.mock/packages/ninja/package.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class Ninja(Package):
```

Comment:
This can probably just be a minimal package, we don't need to copy the entire thing